### PR TITLE
difft: add Spanish translation

### PR DESCRIPTION
**Pull request by kant**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Review by kbdharun — APPROVED** [review]

File: pages.es/common/difft.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# difft
+
+> Compara archivos o directorios basándose en la sintaxis del lenguaje de programación.
```
Comment (tricantivu) [inline pages.es/common/difft.md:3]:
```suggestion
> Compara archivos o directorios en base de la sintáxis del lenguaje de programación.
```

File: pages.es/common/difft.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+> Vea también: `delta`, `diff`.
+> Más información: <https://difftastic.wilfred.me.uk/introduction.html>.
+
+- Compara dos archivos o directorios:
+
+`difft {{ruta/al/archivo_o_directorio1}} {{ruta/al/archivo_o_directorio2}}`
+
+- Informa únicamente la presencia de diferencias entre los archivos:
```
Comment (tricantivu) [inline pages.es/common/difft.md:11]:
```suggestion
- Informa únicamente diferencias entre los archivos:
```

File: pages.es/common/difft.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+- Activa/desactiva el resaltado sintáctico del código fuente (por defecto está activado):
+
+`difft --syntax-highlight {{on|off}} {{ruta/al/archivo1}} {{ruta/al/archivo2}}`
+
+- No muestra nada si no hay diferencias entre los archivos:
+
+`difft --skip-unchanged {{ruta/al/archivo_o_directorio1}} {{ruta/al/archivo_o_directorio2}}`
+
+- Imprime todos los lenguajes de programación admitidos por la herramienta, junto con sus extensiones:
```
Comment (tricantivu) [inline pages.es/common/difft.md:31]:
```suggestion
- Lista todos los lenguajes de programación soportados por la herramienta, junto con sus extensiones:
```

File: pages.es/common/difft.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+- Especifica el modo de visualización (por defecto es `side-by-side`):
+
+`difft --display {{side-by-side|side-by-side-show-both|inline|json}} {{ruta/al/archivo1}} {{ruta/al/archivo2}}`
+
+- Ignora comentarios al comparar:
+
+`difft --ignore-comments {{ruta/al/archivo1}} {{ruta/al/archivo2}}`
+
+- Activa/desactiva el resaltado sintáctico del código fuente (por defecto está activado):
```
Comment (tricantivu) [inline pages.es/common/difft.md:23]:
```suggestion
- Activa o desactiva el resaltado sintáctico del código fuente (por defecto está, activado):
```

File: pages.es/common/difft.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+`difft --syntax-highlight {{on|off}} {{ruta/al/archivo1}} {{ruta/al/archivo2}}`
+
+- No muestra nada si no hay diferencias entre los archivos:
```
Comment (tricantivu) [inline pages.es/common/difft.md:27]:
```suggestion
- Silenciosamente salta los archivos que no hayan cambiado:
```

The previous description did not sound right, what do you think @kant?

File: pages.es/common/difft.md (hunk 4 above)
Comment (kant) [reply pages.es/common/difft.md:23]: please fix that lasting `,`

File: pages.es/common/difft.md (hunk 5 above)
Comment (kant) [reply pages.es/common/difft.md:27]: @tricantivu Silenciosamente saltea los archivos que no hayan cambiado:

File: pages.es/common/difft.md (hunk 1 above)
Comment (kant) [reply pages.es/common/difft.md:3]: sintáxis is not right, _src_: https://dle.rae.es/sintaxis

File: pages.es/common/difft.md (hunk 4 above)
Comment (tricantivu) [reply pages.es/common/difft.md:23]: Ah, meant to put it after `por defecto`. How's that?

Comment (kant) [reply pages.es/common/difft.md:23]: without `,` is perfect

File: pages.es/common/difft.md (hunk 5 above)
Comment (tricantivu) [reply pages.es/common/difft.md:27]: Just now found a another verb, `omitir`.

Maybe `Silenciosamente omite...`?

File: pages.es/common/difft.md (hunk 1 above)
Comment (tricantivu) [reply pages.es/common/difft.md:3]: My bad, I'll correct it now.

File: pages.es/common/difft.md (hunk 5 above)
Comment (kant) [reply pages.es/common/difft.md:27]: Perfect

Comment (tricantivu) [reply pages.es/common/difft.md:27]: Done

**Review by tricantivu — APPROVED** [review]

**Pull request merged**